### DISCUSSION TITLE
https://www.spsanderson.com/steveondata/posts/2024-11-06/

### DISCUSSION THREAD

**Discussion by giscus** [spsanderson/steveondata#16] [Comments]
# https://www.spsanderson.com/steveondata/posts/2024-11-06/

Learn how to effectively use the dollar sign ($) operator in R programming to access data frame columns and list elements. Perfect guide for R beginners with practical examples.

https://www.spsanderson.com/steveondata/posts/2024-11-06/

**Comment by planewryter** [comment]
Commendations, Steve, for the excellent & concise refresher on R's powerful '$'! 
As one who has been programming with R for well over a decade, I found your article to be a great refresher regarding what has become 'muscle memory' (as in: I write the code using well-used syntax patterns; not necessarily with full recollection as to why I apply a specific pattern). 
Looking forward to reading more of your useful posts!
With thanks, Plane Wryter

**Reply by spsanderson** [reply]
Thank you for reading! I'm glad you found the article useful and refreshing. I do try to write consistently Monday through Friday, with Monday, Tuesday and Thursday being R
Wednesday being C (I got turned around this week) and
Friday being Linux.

I am chronicling my journey learning C and Linux so mistakes may occur and feedback is always appreciated.

Have a great day!
